packages/core/src/command/consolidated/consolidated-create.ts

```diff
@@ -21,6 +23,7 @@ const defaultS3RetriesConfig = {
 export type ConsolidatePatientDataCommand = {
   cxId: string;
   patientId: string;
+  patient: Patient;
```

Comment:
Can we remove the `patientId` from the params, since we are now passing the patient, please?